client/components/TestRun/index.jsx

```diff
@@ -814,6 +835,7 @@ const TestRun = () => {
                 <TestNavigator
                     show={showTestNavigator}
                     tests={tests}
+                    testPlanRunId={parseInt(testPlanRunId)}
```

Comment:
I'm not a fan of this explicit int typecasting; why is this not addressed by the routing regular expression?